src/ahbicht/content_evaluation/evaluators.py

```diff
@@ -53,6 +56,9 @@ def __init__(self):
             match = Evaluator._evaluation_method_name_pattern.match(candidate[0])
             if match:
                 self._evaluation_methods[match.groupdict()["condition_key"]] = candidate[1]
+        self.logger.info(
+            "Instantiated %s and found %i evaluation methods", self.__class__.__name__, len(self._evaluation_methods)
```

Comment:
is it not possible to use f-strings in logging?

Reply:
It's rejected by pylint:
https://pylint.pycqa.org/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html
background: creating the final log message from the template is deferred as long as possible:
https://docs.python.org/3/howto/logging.html#optimization

More (easily understandable) background:
* https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation
* https://klichx.dev/2019/11/09/f-strings-and-python-logging/

This is not python-specific btw. Prefering log-templates over string interpolation in log-messages is also recommended for .NET (and e.g. suggested by rider/resharper).